src/metabase/api/activity.clj

```diff
@@ -113,14 +114,18 @@
         "dashboard" [Dashboard
                      :id :name :collection_id :description
                      :archived :is_app_page
+                     [(db/qualify App :id) :app_id]
                      (db/qualify Collection :authority_level)]
         "table"     [Table
                      :id :name :db_id
                      :display_name :initial_sync_status
                      :visibility_type])
-      (cond-> {:where [:in (db/qualify (symbol (str/capitalize model)) :id) ids]}
-        (not= model "table")
-        (merge {:left-join [Collection [:= (db/qualify Collection :id) :collection_id]]}))))
+      (let [model-class (symbol (str/capitalize model))
```

Comment:
nit: Would renaming this to `model-symbol` or `model-symb` improve readability? Toucan has a convention of using `model-symb` internally.